SparkPost/2web2ui
# FE-1422 - Removes autofocus attribute causing error to render on the SSO page

[FE-1422](https://sparkpost.atlassian.net/browse/FE-1422)

### What Changed

- Resolves bug on the SSO page by removing the `autofocus` attribute on the email input (this was an a11y bug as well)

### How To Test

- Visit `/auth/sso` and verify the "Required" error no longer renders on page load

### To Do
